modules/compute/eks/modulestack.yaml

```diff
@@ -35,12 +35,12 @@ Resources:
             Action:
               - "s3:ListBucket"
             Resource:
-              - "arn:aws:s3:::*-dockerimages-replications-*"
+              - "arn:aws:s3:::*-dkr-img-rep-*"
           - Effect: Allow
             Action:
               - "s3:GetObject"
             Resource:
```

Comment:
Can't do this....you need to support backward compatibility, so both versions of the bucket name need to be in the policy, else users will be hosed..

Reply:
Good catch!